client/config/browserstack.config.js

```diff
@@ -30,7 +30,8 @@ const bstackOptions = {
   userName: user,
   accessKey: key,
   // This requires that BrowserStackLocal is running!
-  local: "true"
+  local: "true",
+  seleniumVersion: "3.5.2"
```

Comment:
Why this particular version? Why not the latest (4.1.2)? (see *browserstack.seleniumVersion* on https://www.browserstack.com/automate/capabilities )